src/api/ocpi/v2_2/sessions.controller.ts

```diff
@@ -45,8 +45,21 @@ export class SessionsController extends CustomisableController {
                 }
             })
 
-        }
+            /**
+             * PATCH session
+             */
+            router.put("/receiver/2.2/sessions/:country_code/:party_id/:session_id", async (req, res, next) => {
+                try {
+                    pluggableAPI.sessions!.receiver!.patch(req.body)
```

Comment:
As mentioned in https://github.com/energywebfoundation/rebeam-monorepo/pull/43, we need to pass in the url path parameters into this patch function.